src/main/java/ru/octol1ttle/flightassistant/alerts/nav/UnloadedChunkAlert.java

```diff
@@ -0,0 +1,57 @@
+package ru.octol1ttle.flightassistant.alerts.nav;
+
+import net.minecraft.client.font.TextRenderer;
+import net.minecraft.client.gui.DrawContext;
+import net.minecraft.text.MutableText;
+import net.minecraft.text.Text;
+import org.jetbrains.annotations.NotNull;
+import ru.octol1ttle.flightassistant.HudComponent;
+import ru.octol1ttle.flightassistant.alerts.AlertSoundData;
+import ru.octol1ttle.flightassistant.alerts.BaseAlert;
+import ru.octol1ttle.flightassistant.alerts.IECAMAlert;
+import ru.octol1ttle.flightassistant.computers.AirDataComputer;
+import ru.octol1ttle.flightassistant.computers.safety.ChunkStatusComputer;
+import ru.octol1ttle.flightassistant.computers.safety.ChunkStatusComputer.ChunkStatusLevel;
+import ru.octol1ttle.flightassistant.config.FAConfig;
+
+public class UnloadedChunkAlert extends BaseAlert implements IECAMAlert {
+
+    private final AirDataComputer adc;
+    private final ChunkStatusComputer chunkStatus;
+
+    public UnloadedChunkAlert(AirDataComputer adc, ChunkStatusComputer chunkStatus) {
+        this.adc = adc;
+        this.chunkStatus = chunkStatus;
+    }
+
+    @Override
+    public boolean isTriggered() {
+        if (!adc.isFlying()) {
+            return false;
+        }
+
+        return chunkStatus.getStatus().isAtLeastEqual(ChunkStatusLevel.ADVISORY);
+    }
+
+    @Override
+    public int render(TextRenderer textRenderer, DrawContext context, int x, int y, boolean highlight) {
+        if (FAConfig.computer().chunkStatusWarning.screenDisabled()) {
+            return 0;
+        }
+
+        final MutableText alertText = Text.translatable("alerts.flightassistant.unloaded_chunk");
+        HudComponent.drawText(textRenderer, context, alertText, x, y, chunkStatus.getIndicator(ChunkStatusLevel.CAUTION));
+
+        final int textWidth = textRenderer.getWidth(alertText) + 5;
+        final String seconds = "%.1f".formatted(chunkStatus.getLastDiffMS() * 0.001f);
+        final Text text = Text.translatable("alerts.flightassistant.unloaded_chunk_time", seconds);
+        HudComponent.drawHighlightedText(textRenderer, context, text, x + textWidth, y, chunkStatus.getIndicator(ChunkStatusLevel.WARNING), highlight);
+        return 1;
+    }
+
+    @Override
+    public @NotNull AlertSoundData getSoundData() {
+        return AlertSoundData.ifEnabled(FAConfig.computer().chunkStatusWarning, AlertSoundData.MASTER_CAUTION);
```

Comment:
Alerts in red shouldn't be combined with the MASTER CAUTION sound (and CAUT doesn't seem appropriate for how dangerous flying in unloaded chunks can be)